.github/workflows/action-test-kind.yml

```diff
@@ -19,16 +19,11 @@ jobs:
       - name: Setup build env
         run: ./scripts/setup-linux.sh
         shell: bash
-      - name: Download artifacts
-        uses: actions/download-artifact@master
-        with:
-          name: containerd-shim-${{ inputs.runtime }}-${{ inputs.os }}
-          path: dist
-      - name: Unpack artifats
+      - name: build specific artifacts
         shell: bash
         run: |
           mkdir -p dist/bin
-          tar -xzf dist/containerd-shim-${{ inputs.runtime }}-${{ inputs.os }}.tar.gz -C dist/bin
+          docker build -o dist/bin/ .
```

Comment:
This isn't really what we want, it was just a way to test the correctly linked glibc binary works